tests/Toolkit/VTest.php

```diff
@@ -600,18 +600,18 @@ public function testSize(): void
 		$this->assertFalse(V::size(7, 5, '<'));
 
 		$field = new Field(null, 'foo', 2);
-		$this->assertTrue(V::size($field->foo(), 2));
-		$this->assertTrue(V::size($field->foo(), 3, '<'));
-		$this->assertTrue(V::size($field->foo(), 1, '>'));
-		$this->assertFalse(V::size($field->foo(), 1, '<'));
-		$this->assertFalse(V::size($field->foo(), 3, '>'));
+		$this->assertTrue(V::size($field, 2));
+		$this->assertTrue(V::size($field, 3, '<'));
+		$this->assertTrue(V::size($field, 1, '>'));
+		$this->assertFalse(V::size($field, 1, '<'));
+		$this->assertFalse(V::size($field, 3, '>'));
```

Comment:
`Explanation:` I corrected these, as calling the non-existent field method `$field->foo()` would just return `$field` anyways. `V::size()`  itself takes care of applying the `->value()` field method.